### PR TITLE
update to v0.11

### PR DESCRIPTION
**Pull request by daltonbohning**
Update to version 0.11 and comment out patch for now.

PR-repos: dtcmp@PR-3

Signed-off-by: Dalton Bohning <daltonx.bohning@intel.com>

### CODE REVIEW

**Comment by daltonbohning** [comment]
Merge order for this PR and dependencies:
- (VERIFYING) Backwards compatibility - https://github.com/daos-stack/daos/pull/4511
- lwgrp https://github.com/daos-stack/lwgrp/pull/3
- dtcmp (depends on lwgrp) https://github.com/daos-stack/dtcmp/pull/3
- This PR (depends on dtcmp)
  - Verify current mpifileutils works in CI: https://github.com/daos-stack/daos/pull/4511
  - Verify THIS mpifileutils works in CI: https://github.com/daos-stack/daos/pull/4382

**Comment by daltonbohning** [comment]
@brianjmurrell Functionally, everything appears to be working - just waiting for daos-stack/daos#4511 to be approved and merged.
We'll want to wait **_after_** daos-stack/daos#4511 is merged before landing any of these package PRs. (See comment above)

**Comment by brianjmurrell** [comment]
> @brianjmurrell Functionally, everything appears to be working - just waiting for [daos-stack/daos#4511](https://github.com/daos-stack/daos/pull/4511) to be approved and merged.
> We'll want to wait **_after_** [daos-stack/daos#4511](https://github.com/daos-stack/daos/pull/4511) is merged before landing any of these package PRs. (See comment above)

The wait (between landing daos-stack/daos#4511 and landing this) needs to include an announcement that a master merge is necessary for all new PR submissions and a _reasonable_ time window has elapsed to allow PRs in flight (i.e. those currently running) to complete.

To determine that wait time, you can make a record of the job that is at the back of the queue when (or very shortly after) you make your announcement (which only happens after daos-stack/daos#4511 has landed) and once that job has completed, it can be assumed that everything else in the queue after that has merged and has the changes.  Then this can land.

File: mpifileutils.spec
Comment (brianjmurrell) [inline mpifileutils.spec:47]: When the `Version:` is incremented, the `Release:` goes back to `1`.
```suggestion
Release:	1%{?relval}%{?dist}
```

File: mpifileutils.spec
Comment (brianjmurrell) [inline mpifileutils.spec:54]: So this PR is to use upstream's official 0.11 release, without any patch from master?

If so, you can remove https://github.com/daos-stack/mpifileutils-pkg/blob/ab37048e0e0c1417d66ae422c4d11898bc3fa137/Makefile#L4-L6 so that this PR doesn't have a `.g<git-hash>` in the `Release:`.
```suggestion
```

File: mpifileutils.spec
Comment (brianjmurrell) [inline mpifileutils.spec:201]:
```suggestion
* Mon Feb 01 2021 Dalton A. Bohning <daltonx.bohning@intel.com> - 0.11-1
```

**Review by brianjmurrell — CHANGES_REQUESTED** [review]

**Comment by daltonbohning** [comment]
> The wait (between landing [daos-stack/daos#4511](https://github.com/daos-stack/daos/pull/4511) and landing this) needs to include an announcement that a master merge is necessary for all new PR submissions and a _reasonable_ time window has elapsed to allow PRs in flight (i.e. those currently running) to complete.
> 
> To determine that wait time, you can make a record of the job that is at the back of the queue when (or very shortly after) you make your announcement (which only happens after [daos-stack/daos#4511](https://github.com/daos-stack/daos/pull/4511) has landed) and once that job has completed, it can be assumed that everything else in the queue after that has merged and has the changes. Then this can land.

Sure, that all makes sense! Luckily we don't have any datamover,pr tests. I'll give it a few days after merging [daos-stack/daos#4511](https://github.com/daos-stack/daos/pull/4511) just to be safe. (Not a high priority)
I'll check back here after that time.
Thank you!

File: mpifileutils.spec
Comment (daltonbohning) [reply mpifileutils.spec:54]: We _**will**_ be building against another patch version in the future, since  we are still actively making changes.
In this case, should I still remove the `.g<git-hash>` entirely? Or maybe just comment it out for now?

File: mpifileutils.spec
Comment (daltonbohning) [reply mpifileutils.spec]: I think I will also need to update daos-stack/daos#4511 and daos-stack/lwgrp#3 in the same manner.

**Comment by brianjmurrell** [comment]
> Luckily we don't have any datamover,pr tests.

Hrm.  Well, in that case, I don't see any need to wait.  Why don't we do any datamover tests in PRs?  I guess it's expected that `Features: datamover` be used for such changes.  Fair enough.

> I'll give it a few days after merging [daos-stack/daos#4511](https://github.com/daos-stack/daos/pull/4511) just to be safe. (Not a high priority)

As you wish.  But if there are actually no datamover tests in PRs the need to wait goes away.  Ahh.  But are there any in `daily_regression`?  If so, we just need to make sure not to land the mpifileutils PR during a daily_regression run that started before this lands.

File: mpifileutils.spec
Comment (brianjmurrell) [reply mpifileutils.spec:54]: You can leave the `%{?relval}` in `Release:	1%{?relval}%{?dist}` if you want, but you need to remove (or I guess comment out) the the lines in the `Makefile`.  But you should also remove the reference to the `#Patch1:` line and restore it again, when a patch is being used.

**Comment by daltonbohning** [comment]
> > Luckily we don't have any datamover,pr tests.
> 
> Hrm. Well, in that case, I don't see any need to wait. Why don't we do any datamover tests in PRs? I guess it's expected that `Features: datamover` be used for such changes. Fair enough.

At one point we had some `datamover,pr` tests, but they were moved to `daily_regression`

> > I'll give it a few days after merging [daos-stack/daos#4511](https://github.com/daos-stack/daos/pull/4511) just to be safe. (Not a high priority)
> 
> As you wish. But if there are actually no datamover tests in PRs the need to wait goes away. Ahh. But are there any in `daily_regression`? If so, we just need to make sure not to land the mpifileutils PR during a daily_regression run that started before this lands.

Right. If [daos-stack/daos#4511](https://github.com/daos-stack/daos/pull/4511) lands today or tomorrow, I'll probably wait until Monday anyway _**just in case**_ so something doesn't break over the weekend. I.e. someone using `Features: datamover` in a PR.

**Comment by brianjmurrell** [comment]
> I'll probably wait until Monday anyway _**just in case**_ so something doesn't break over the weekend. I.e. someone using `Features: datamover` in a PR.

Heh, yeah.  I did thing about the situation of somebody using `Features: datamover` in a PR also.  Probably best to be safe/conservative.

File: mpifileutils.spec
Comment (daltonbohning) [reply mpifileutils.spec]: Sure! Just pushed

**Review by brianjmurrell — APPROVED** [review]

**Pull request merged**
